manual/src/Makefile

```diff
@@ -137,7 +137,8 @@ warnings-help.etex: $(SRC)/utils/warnings.ml $(SRC)/ocamlc
 	 $(SET_LD_PATH) $(SRC)/boot/ocamlrun $(SRC)/ocamlc -warn-help \
 	 | LC_ALL=C sed -e 's/^ *\([0-9][0-9]*\) *\[\([a-z][a-z-]*\)\]\(.*\)/\\item[\1 "\2"] \3/' \
                -e 's/^ *\([0-9A-Z][0-9]*\) *\([^]].*\)/\\item[\1] \2/'\
-	) >$@
+	)\
+         | sed -e 's/@/\\@/g' >$@
```

Comment:
out of curiosity, is there a reason not to pipe inside the parenthesis, to have a pipe sequence with the previous one?

Reply:
Not really, I was thinking of splitting this small fix from the previous rewrite.